milestone1/src/main/java/pt/ulisboa/tecnico/milestone1/controller/UserController.java

```diff
@@ -0,0 +1,26 @@
+package pt.ulisboa.tecnico.milestone1.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import pt.ulisboa.tecnico.milestone1.domain.User;
+import pt.ulisboa.tecnico.milestone1.repository.UserRepository;
+
+@RestController
+@RequestMapping("/user")
+public class UserController {
+    @Autowired
+    private UserRepository userRepository;
+
+    @GetMapping("/{id}")
```

Comment:
This looks like REST... Why do we need this? Will the server execute a REST call to the database and the database insert the data?

We need a middleware between the server and the database. Where is that class?

Reply:
This code was made to be able to test the interaction with the database.
But i have already done the server part that communicates with the database (i will commit the changes after the meeting for further review).
The protocol between client and server that i chose was REST (we'll discuss today in our meeting).